<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetInput</class>
 <widget class="QDialog" name="SetInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3" rowspan="3" colspan="2">
    <widget class="QWidget" name="fileInformation_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="fileInformation_label">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QWidget" name="data_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="data_groupBox">
        <property name="title">
         <string>Data</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="1">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QLineEdit" name="filename_lineEdit"/>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QTableWidget" name="data_tableWidget"/>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="searchFile_pushButton">
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QWidget" name="modification_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="modification_groupBox">
        <property name="title">
         <string>Modification</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="3" column="0">
          <widget class="QPushButton" name="deleteRows_pushButton">
           <property name="text">
            <string>Delete Rows</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QPushButton" name="deleteColumns_pushButton">
           <property name="text">
            <string>Delete Columns</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QPushButton" name="saveFile_pushButton">
           <property name="text">
            <string>Save CSV File</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QWidget" name="load_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_7"/>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QWidget" name="close_widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="close_pushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>data_tableWidget</tabstop>
  <tabstop>filename_lineEdit</tabstop>
  <tabstop>deleteRows_pushButton</tabstop>
  <tabstop>deleteColumns_pushButton</tabstop>
  <tabstop>saveFile_pushButton</tabstop>
  <tabstop>close_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>close_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>SetInput</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>479</x>
     <y>501</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
